app/src/main/java/org/p2p/wallet/push_notifications/ineractor/PushNotificationsInteractor.kt

```diff
@@ -35,7 +42,13 @@ class PushNotificationsInteractor(
             clientId = tokenKeyProvider.publicKey
         )
 
-        return deviceTokenRepository.sendDeviceToken(deviceToken)
+        try {
+            deviceTokenRepository.sendDeviceToken(deviceToken)
+        } catch (e: ServerException) {
+            Timber.e(e, "Error sending device token $token to server")
+            delay(TOKEN_SEND_RETRY_DELAY_MS)
```

Comment:
It's not a good idea to block thread resources for one minute, i think it's insufficient way and we need to go with another way of implementation

Reply:
Let's not overhead the solution for now and check how it performs on the QA stage